### ISSUE TITLE
4-6th Split Placements Showing Too High

### ISSUE THREAD

**Issue by MuteD6** [andrewhuntsmith/LiveSplit.PacePlacement#2]
In runs over many attempts, the 4th split is commonly shown as being 1st, 2nd each time, which would be inaccurate. I believe the same is happening across the 5th and 6th splits, other splits are unclear. Thanks.

**Comment by andrewhuntsmith** [comment]
There are a few things that could cause this. Can you send me your splits file? If I have that, I can try to narrow down where the issue is coming from.

**Comment by MuteD6** [comment]
Absolutely, here you go.These are the files my friends have that seems to
be causing the problem. These splits are a bit outdated but I hope they
help. Thanks!

**Comment by MuteD6** [comment]
Absolutely, I have replied to the e-mail chain. Thanks so much for your help

**Comment by andrewhuntsmith** [comment]
I'm not seeing a file attachment, here or on the e-mail chain. Can you double check that you attached a file? I don't use github often, so I also might be missing it.

**Comment by MuteD6** [comment]
Github isn't accepting Lsl files, I'm wondering if I could send them to you
personally?

**Comment by andrewhuntsmith** [comment]
That would be fine. You can email them to me, my email address is the same as my username here @gmail. Alternately, you can send them to me on discord, where my username is xandotoaster.
